src/services/account/useAccountDetails.ts

```diff
@@ -77,7 +77,7 @@ export const SubscriptionDetailSchema = z
     currentPeriodEnd: z.number(),
     customer: z
       .object({
-        discount: z.number(),
```

Comment:
this is unrelated but is a regression I need to merge with one of my upcoming PRs